### ISSUE TITLE
get_library_albums returns incorrect keys

### ISSUE THREAD

**Issue by mrgou** [sigma67/ytmusicapi#84]
I noticed that the `get_library_albums` method returns data mapped to incorrect keys:

```json
{
  "artists": "Album",
  "browseId": "MPREb_TcjAa6Gjdop",
  "thumbnails": [
    {
      "height": 226,
      "url": "https://lh3.googleusercontent.com/ZVOHvYxy66tBHPDD4DXnypgIC7d6FSuXs706ePGrv2U7rOU2tFjbnyEGTcfmy1PbKcZwMJnRS6NAguI9=w226-h226-l90-rj",
      "width": 226
    },
    {
      "height": 544,
      "url": "https://lh3.googleusercontent.com/ZVOHvYxy66tBHPDD4DXnypgIC7d6FSuXs706ePGrv2U7rOU2tFjbnyEGTcfmy1PbKcZwMJnRS6NAguI9=w544-h544-l90-rj",
      "width": 544
    }
  ],
  "title": "Evolution Ending",
  "trackCount": "1998",
  "type": "Album",
  "year": "Naked Funk"
}
```
As you can see, `artists` is set to the value of `type`, then `trackCount` and `year` are shifted, with the data of the former actually missing.

**Comment by sigma67** [comment]
It seems YouTube has changed the way these are displayed, causing a mismatch in key assignment.

**Comment by mrgou** [comment]
Will you fix this in a next release?

**Comment by sigma67** [comment]
I'll do a new release by the end of this week, just to see if anything else pops up by then.

**Comment by mrgou** [comment]
Thank you! :-)

**Comment by sigma67** [comment]
Released v0.10.1

bdbaada
